addons/zylann.scene_code_converter/converter.gd

```diff
@@ -170,7 +170,7 @@ static func _value_to_code(v) -> String:
 		TYPE_COLOR:
 			return str("Color(", v.r, ", ", v.g, ", ", v.b, ", ", v.a, ")")
 		TYPE_STRING:
-			return str("TTR(\"", v, "\")")
+			return str("TTR(\"", v.replace('"', '\\"'), "\")")
```

Comment:
```suggestion
			return str("TTR(\"", v.c_escape(), "\")")
```
https://docs.godotengine.org/en/stable/classes/class_string.html#class-string-method-c-escape

Reply:
Hmm, I wonder why I haven't found this method earlier, thanks!